tests/modules/programs/zellij/disable-shells.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, ... }:
+
+{
+  programs = {
+    zellij = {
+      enable = true;
+      enableBashIntegration = false;
+      enableZshIntegration = false;
+      enableFishIntegration = false;
+    };
+    bash.enable = true;
+    zsh.enable = true;
+    fish.enable = true;
+  };
+
+  # Needed to avoid error with dummy fish package.
+  xdg.dataFile."fish/home-manager_generated_completions".source =
+    lib.mkForce (builtins.toFile "empty" "");
+
+  test.stubs = {
+    zsh = { };
+    zellij = { };
+  };
+
+  nmt.script = ''
+    assertFileNotRegex home-files/.bashrc \
+      'eval "$(zellij setup --generate-auto-start bash)"'
+
+    assertFileNotRegex home-files/.zshrc \
+      'eval "$(zellij setup --generate-auto-start zsh)"'
+
+    assertFileNotRegex home-files/.config/fish/config.fish '@zellij@'
```

Comment:
I still feel like this isn't a very useful test... If the integration snippets ever change, this test won't fail, so it probably won't be updated, and then it won't be testing anything any more.

Just remove it IMO.

Reply:
Talking about the fish one in particular? Or the lot of them? Both the tests for enabling and disabling? 

We can also make the fish regex look for `eval (zellij setup --generate-auto-start fish | string collect)`.

Reply:
As eluded to in a past comment, I ripped these tests off of another package doing this exact thing. Not at all married to them at all. Just let me know if by getting rid of the tests, you mean all of the tests introduced by this PR or just some subset (like the disable fish one?)

Reply:
I'm talking about the `disable-shells.nix` test.

Reply:
Sounds good. I'll drop those.